Assets/Scripts/Dodo.cs

```diff
@@ -238,6 +239,11 @@ private void Move()
                 MoveTowardMountain();
                 break;
         }
+
+        if (_isInMud && _wc.getWorldSpeedPercentage() > dodoInMudSpeed)
```

Comment:
Not the _most_ ideal, but there's a lot of places that set the speed so I figured a final check to make sure it's not too high should work OK enough